packages/server/.env.test

```diff
@@ -0,0 +1 @@
+SERVER_NETWORK_ID=123456789
```

Comment:
We should probably rename this at some point, since `SERVER` is pretty ambiguous.

Reply:
Agreed that `SERVER` is ambiguous. `SERVER_NETWORK_ID` can be Ganache, Ropsten, Mainnet, etc. So maybe `CHAIN_NETWORK_ID`?